packages/skeleton-react/src/components/Accordion/Accordion.test.tsx

```diff
@@ -0,0 +1,179 @@
+import { describe, expect, it } from "vitest";
+import { render, waitFor } from "@testing-library/react";
+
```

Comment:
You can also test the functionality of the accordion with the `act()` function similarly to this:
```ts
import { act } from '@testing-library/svelte';

// describe > it > ...
await act(async () => {
    // user selects the first input
    await userEvent.click(inputElements[0]);
    // user types a number
    await userEvent.type(inputElements[0], '1');
});
// expect something to have changed
```
I'd like to see a test like this

Reply:
This would use `fireEvent.click()` accordion to the Testing Library docs:
https://testing-library.com/docs/react-testing-library/example-intro#full-example

I actually had an example of this, but the way the component works in React clicking updates the Context API state. But nothing I tried would actually modify the DOM state related to the Context values. I spent half the afternoon on this alone heh!

Learning how to test React's Content Provider is completely new to me, so it's a bit of a black box. But I definitely want to implement something along these lines!

Reply:
If I remember and can find the time I'll have a look at it tomorrow too. Maybe an additional pair of eyes can help